src/containers/Review/assignment/useGetAssignmentOptions.ts

```diff
@@ -57,15 +63,22 @@ const useGetAssignmentOptions = () => {
 
     const assigneeOptions = {
       selected: previousAssignee || NOT_ASSIGNED,
+      isSubmitted: false,
       isCompleted: false,
       options: [...currentUserAssignable.map((assignment) => getOptionFromAssignment(assignment))],
     }
 
-    assigneeOptions.options.push({
-      key: NOT_ASSIGNED,
-      value: NOT_ASSIGNED,
-      text: strings.ASSIGNMENT_NOT_ASSIGNED,
-    })
+    // Move current user "Yourself" to top of list
+    const currentUserOption = assigneeOptions.options.find(
+      (option) => option.value === currentUser?.userId
+    )
+
+    // Remove currentUser from the list and add at the top
+    const sortedOptions = assigneeOptions.options.filter(
+      (option) => option.key !== currentUser?.userId
+    )
+    if (currentUserOption) sortedOptions.unshift(currentUserOption)
+    assigneeOptions.options = sortedOptions
```

Comment:
Suggestion you gave @CarlosNZ 

The missing part (to remove the duplicate user) was this:
```
const sortedOptions = assigneeOptions.options.filter(
      (option) => option.key !== currentUser?.userId
    )
```